stable/magento/templates/externaldb-secrets.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if not .Values.mariadb.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-%s" .Release.Name "externaldb"  }}
+  labels:
+    app: {{ printf "%s-%s" .Release.Name "externaldb"  }}
```

Comment:
this should be `app: {{ template "magento.fullname" . }}` isn't it? Also if we make this change then you really don't need to have the `chart`, `release` and `heritage` labels.

Reply:
yep the app label should be `app: {{ template "magento.fullname" . }}` but why the rest of labels are not needed? It also applies to the other secret? 
https://github.com/tompizmor/charts/blob/existing_database_magento/stable/magento/templates/secrets.yaml#L7-L9